src/main/webapp/dev/js/main/instructorCourseEdit.es6

```diff
@@ -529,4 +530,8 @@ $(document).ready(() => {
     $(document).on('click', '.view-role-details', (e) => {
         showInstructorRoleModal($(e.target).data('role'));
     });
+
+    $('.display-to-student').change((e) => {
+        $('.title-displayed').prop('disabled', !e.target.checked);
```

Comment:
Upon checking the behaviour again, it appears that this toggles the `disabled` state of every "display to students as" checkbox, including the instructors that are not being edited, which may confuse the user. Ideally, the checkbox should only toggle the state of the input field for the same instructor that is currently being added/edited.